hummingbot/wallet/ethereum/watcher/new_blocks_watcher.py

```diff
@@ -99,6 +99,7 @@ def block_number(self) -> int:
 
     async def fetch_new_blocks_loop(self):
         last_timestamp_received_blocks: float = 0.0
+        block_hash = ""
```

Comment:
not needed.

Reply:
block_hash is used in the below exception handling, and may not be defined if the exception is thrown early.
https://github.com/CoinAlpha/hummingbot/blob/ec8ff10f51711d9b161f23aaf9df5d78434cda5b/hummingbot/wallet/ethereum/watcher/new_blocks_watcher.py#L138

Reply:
ok then